Mailer/app/api/Validation/InputValidatorTrait.php

```diff
@@ -29,17 +32,23 @@ protected function validateInput(string $request, string $schema)
         $schema = file_get_contents(__DIR__ . '/schemas/' . $schema);
         $data = Json::decode($request);
         $validator = new Validator();
-        $validator->validate($data, (object) $schema);
+        $validator->validate($data, (object) json_decode($schema));
```

Comment:
Nette has it's own `Json` helper class for decoding. It's alredy being imported here. Please use that.